### PR TITLE
feat: widget docs

### PR DESCRIPTION
**Pull request by shoom3301**
# Description

Copy-pasted https://github.com/cowprotocol/cowswap/blob/develop/libs/widget-lib/docs/README.md

image

### CODE REVIEW

**Comment by shoom3301** [comment]
I have read the CLA Document and I hereby sign the CLA

File: docs/front-end/widget/README.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
+# CowSwap Widget
```
Comment (alfetopito) [inline docs/front-end/widget/README.md:1]:
```suggestion
# CoW Swap Widget
```

File: docs/front-end/widget/README.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+Integrate the power of `CowSwap` into your product!
```
Comment (alfetopito) [inline docs/front-end/widget/README.md:3]: Marketing will kill you. Use capital `W` :D

```suggestion
Integrate the power of `CoW Swap` into your product!
```

**Review by alfetopito — APPROVED** [review]
Looks great!
Just nitpicts regarding the branding

File: docs/front-end/widget/README.md (hunk 1 above)
Comment (shoom3301) [reply docs/front-end/widget/README.md]: Thank you!
Fixed everywhere: https://github.com/cowprotocol/docs-v2/pull/25/commits/96928b27ce5d6bf689ef7182af43e5d609b3fd05

**Comment by mfw78** [comment]
Please kindly don't merge this yet. There will be rebasing coming.

**Review by anxolin — COMMENTED** [review]
This is great. Just a question. Is it worth it to, reduce duplication by WGETing or sth the Markdown from the widget? (or, if we are doing the git submodules approach already for TS generation, we could already get the content from the dependency)

EDIT: Actually, if we use git submodules already for generating TS docs, maybe we can just read from the README directly?

**Review by mfw78 — APPROVED** [review]

**Pull request merged**